src/clp_ffi_py/Python.hpp

```diff
@@ -8,3 +8,20 @@
 #define PY_SSIZE_T_CLEAN
 #include <Python.h>
 #include <structmember.h>
```

Comment:
Do you remember why we included `structmember.h` in this file? Was it to try and resolve a clang-tidy warning?

Reply:
I don't remember the exact reason, but it wasn't clang-tidy related

Reply:
Everything seems to compile if we remove it. Let's revisit in #76.

Reply:
`structmember.h` seems to define some macros for CPython. I think we included this header for some safety concern (i.e., ensure some macro is defined), but I don't remember exactly which part of the code requires it. But yeah, let's revisit it in future PRs.